core/src/components/list/test/inset/list.e2e.ts

```diff
@@ -0,0 +1,15 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('list: inset', () => {
+  test.beforeEach(async ({ page }) => {
```

Comment:
Can we skip RTL tests? Also, do we need to run iOS tests? If not, let's skip those too.

Reply:
Talked with Brandy offline. I don't remember why I asked this 😂